hystrix/src/main/java/feign/hystrix/HystrixInvocationHandler.java

```diff
@@ -62,6 +63,8 @@ protected Object run() throws Exception {
 
     if (HystrixCommand.class.isAssignableFrom(method.getReturnType())) {
       return hystrixCommand;
+    } else if (Observable.class.isAssignableFrom(method.getReturnType())) {
+      return hystrixCommand.toObservable();
```

Comment:
There is also `observe()` what has different semantics from `toObservable()`. If  `toObservable()` is what you want, you'll need to document it.